lumicks/pylake/detail/value.py

```diff
@@ -0,0 +1,170 @@
+import math
+
+import numpy as np
+
+
+class ValueMixin:
```

Comment:
I think it would be good to give this class an `__init__` method. It makes it clearer that derived classes should initialize it:
```py
class ValueMixin:
	def __init__(self, value):
	    self.value = value

class Parameter(ValueMixin):
    def __init__(self, value, description):
        super().__init__(value)
        self.description = description
```

You could also consider narrowing down the type a bit here. Maybe:
```py
from numbers import Number

class ValueMixin:
	def __init__(self, value: Number | bool | str):
	    self.value = value
```